README.md

```diff
@@ -1,2 +1,16 @@
-# terminology
-Codesystems, questionnaires, and validation tools for them, for the All of Us project.
+# All of Us Terminology
+
+Codesystems, questionnaires, and validation tools for them, for the All of Us
+project.
+
+## Usage
+
+![workflow diagram](doc/overview.png)
+
+1.  Export questionnaires from Vibrent's authoring tool and place them in
+    `source/`.
+1.  `tools/validate_and_publish [--commit]` writes to `generated/`.
```

Comment:
Maybe this winds up being a short term vs long term thing: in the short term we support "publish from where you are, using the existing POST Questionnaire" mechanism — then in the long term we introduce the "publish the content at this commit" API.

The main scenario I want to avoid is having deploying content to prod with no record of the content in GitHub.